bika/lims/profiles/default/workflows/bika_analysis_workflow/definition.xml

```diff
@@ -187,7 +192,7 @@
       <permission-role>Manager</permission-role>
     </permission-map>
     <permission-map name="BIKA: Edit Field Results" acquired="False">
-      <permission-role>LabClerk</permission-role>
+      <permission-role>Analyst</permission-role>
```

Comment:
In my opinion, an Analyst shouldn't never be able to modify Field results, I think this is a task that must concern to the Sampler only. If we were strict, the scope of work of an Analyst regarding to results introduction should be limited to those analyses performed inside the lab and ideally, only for those to which has been assigned through a Worksheet. Is the responsability of the labman to plan the work and assign the tasks to the available resources. 

Note `to_be_preserved` state is reached before the sample is received (not yet available by personnel from inside the lab). See diagram here: https://github.com/senaite/bika.lims/blob/senaite-integration/docs/resources/bika_sample_workflow.svg

Anyhow, these are the things that will definitely depend on the lab idiosyncrasy, so I suggest that once we have all permissions in place, the changes in permissions should be done in a lab-specific add-on.